### ISSUE TITLE
Nav: Add "Tools" landing page

### ISSUE THREAD

**Issue by tbradsha** [Automattic/wp-calypso#50567]
**What**
When one clicks on the "Tools" top level menu in Nav Unification, the default page is "Marketing and Integrations", which is only a subset of the many tools available. Instead, it should go to a Tools landing page.

**Why**
Clicking a top-level menu and ending up on a page that is only a minor item of the parent menu item is unintuitive.

**Comment by tjcafferkey** [comment]
Thanks for raising @tbradsha. The tools landing page you mentioned which is the parent menu link when nav unification is disabled is still accessible via the Tools > Other Tools menu item

**Comment by mmtr** [comment]
> Clicking a top-level menu and ending up on a page that is only a minor item of the parent menu item is unintuitive.

This is the default behavior of the menu in Core. Clicking on the parent menu will redirect users to the URL of the first submenu.

To solve this we should change the first submenu with a more generic sceen.

**Comment by tbradsha** [comment]
Indeed, in my ideal world the first submenu item would be something other than Marketing.

**Comment by obenland** [comment]
@cpapazoglou Did we add "Other Tools" to provide a way back to My Blogs?

**Comment by mmtr** [comment]
> @cpapazoglou Did we add "Other Tools" to provide a way back to My Blogs?

Yeah, that was the main reason why I added it in https://github.com/Automattic/jetpack/pull/18586, but now that we have a link in "Me" to go to "My Blogs" I was thinking on removing it as part of https://github.com/Automattic/jetpack/pull/19114.

What's your take on that?

**Comment by mmtr** [comment]
We've been actually asked to remove the "Other tools" page (https://github.com/Automattic/wp-calypso/issues/50193) so I guess we'd need to create a different landing page.

**Comment by obenland** [comment]
Can you help me understand what's not working with the current one? Sure, it is framed in a Marketing offering, but it shows a list of tools, like logo creators, CRM, earn potential, etc

**Comment by tbradsha** [comment]
It probably "works as intended", but to me, the "Tools" menu should lead built-in functions, not marketing upsells.
